.github/workflows/owasp.yml

```diff
@@ -0,0 +1,23 @@
+name: OWasp Security Scan
+
+on:
+  workflow_dispatch:
+
+jobs:
+  owasp:
+    name: 'OWASP Test on Staging'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: ZAP Scan
+        uses: zaproxy/action-full-scan@v0.2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          docker_name: 'owasp/zap2docker-stable'
+          target: 'https://s-manage-training-for-early-career-teachers.education.gov.uk/sandbox'
```

Comment:
Soo, what will it test exactly? Just the things discoverable from that page?

Reply:
It is a spider, and that page has links to most places it needs to get to, it may need parallel runs but it is a start. unfortunately I can't run it manually until it is on main branch! so will have to make it on pull for this branch first

Reply:
there is also an API scanner I will look to run in parallel that runs off the openapi spec

https://www.zaproxy.org/docs/docker/api-scan/